toolchains/jp_workspaces/MODULE.bazel.jp61

```diff
@@ -0,0 +1,61 @@
+module(
```

Comment:
Make sure you indicate in the file name that this is a template

Reply:
changed as suggested MODULE.bazel.tmpl.